# UDP通信配置
multicast:
  address: "224.0.1.2"
  port: 9999

# 话题配置
topics:
  car_rank: "/car_rank"
  goal: "/goal"
  lock_wheel: "/lock_wheel"
  estop: "/e_stop"
  correct_xy: "/correct_xy"

# 文件路径配置
files:
  car_name: "~/.config/autofleet/car_name.txt"
  car_rank: "~/.config/autofleet/car_rank.txt"
  correct_xy: "~/.config/autofleet/correctxy.txt"

